src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -83,6 +84,8 @@ public void should_store_a_PropertyAndSchoolAndRoom() {
         assertNotNull(propertySaved.getLotData());
         assertEquals(propertySaved.getLotData(), propertyToSave.getLotData());
 
+        assertNotNull(propertySaved.getCommunity());
```

Comment:
just add an extra assertNotNull(propertySaved.getCommunity().getId()) because we need to be sure that the id was filled. Don't worry about the other id's, send is adding the validation to other in her branch

Reply:
done

Reply:
we still don't have the not null validation for community id

Reply:
sorry added it